modules/dag-l0/src/main/scala/io/constellationnetwork/dag/l0/infrastructure/snapshot/GlobalSnapshotConsensus.scala

```diff
@@ -89,6 +89,7 @@ object GlobalSnapshotConsensus {
       feeCalculator = FeeCalculator.make(feeConfigs)
       snapshotAcceptanceManager = GlobalSnapshotAcceptanceManager.make(
         sharedCfg.fieldsAddedOrdinals.tessellation3Migration.getOrElse(sharedCfg.environment, SnapshotOrdinal.MinValue),
+        sharedCfg.fieldsAddedOrdinals.tessellation301Migration.getOrElse(sharedCfg.environment, SnapshotOrdinal.MinValue),
```

Comment:
reasonable idea to add new ordinals for these sorts of things but if that is required then we may want to plan out and group functionality like with v3 (since I could see this blowing up quickly) though I think that is more an overall strategy that @ryle-goehausen or @AlexBrandes might have opinions on how to approach